src/main/kotlin/uk/gov/justice/hmpps/architecture/model/ProbationAllocationTool.kt

```diff
@@ -0,0 +1,56 @@
+package uk.gov.justice.hmpps.architecture.model
+
+import com.structurizr.model.Container
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.AutomaticLayout
+import com.structurizr.view.ViewSet
+import uk.gov.justice.hmpps.architecture.HMPPSSoftwareSystem
+import uk.gov.justice.hmpps.architecture.annotations.Tags
+
+class ProbationAllocationTool private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+    lateinit var ui: Container
+    lateinit var allocationsApi: Container
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "Probation Case Allocation tool",
+        "Allows users to allocate probation practitioners to PoPs (People on Probation)"
+      )
+
+      ui = system.addContainer("Manage a Workforce UI", "UI for allocating cases", "Node").apply {
+        url = "https://github.com/ministryofjustice/manage-a-workforce-ui"
+      }
+
+      allocationsApi = system.addContainer("Allocations API", "Provides information related to unallocated cases", "Kotlin").apply {
+        url = "https://github.com/ministryofjustice/hmpps-allocations"
```

Comment:
If you tag this as a probation domain API then it will appear on the API views
```suggestion
        url = "https://github.com/ministryofjustice/hmpps-allocations"
        Tags.DOMAIN_API.addTo(this)
        Tags.AREA_PROBATION.addTo(this)
```